homeassistant/components/fritz/config_flow.py

```diff
@@ -34,18 +37,17 @@ class FritzBoxToolsFlowHandler(ConfigFlow, domain=DOMAIN):
 
     VERSION = 1
 
-    def __init__(self):
+    def __init__(self) -> None:
         """Initialize FRITZ!Box Tools flow."""
-        self._host = None
-        self._entry = None
-        self._name = None
-        self._password = None
-        self._port = None
-        self._username = None
-        self.import_schema = None
-        self.fritz_tools = None
-
-    async def fritz_tools_init(self):
+        self._host: str
+        self._entry: ConfigEntry
+        self._name: str
+        self._password: str
+        self._port: int
+        self._username: str
+        self.fritz_tools: FritzBoxTools
```

Comment:
They were `None` but now they will be undefined on the first access. Maybe you just don't need to store them here?
```suggestion
        self._host: str | None = None
        self._entry: ConfigEntry | None = None
        self._name: str | None = None
        self._password: str | None = None
        self._port: int | None = None
        self._username: str | None = None
        self.fritz_tools: FritzBoxTools | None = None
```